### PR TITLE
Renaming ros2controlcli verbs

### PR DESCRIPTION
**Pull request by mahaarbo**
Addresses https://github.com/ros-controls/ros2_control/issues/359, https://github.com/ros-controls/ros2_control/issues/361.

### CODE REVIEW

File: ros2controlcli/README.md [hunk 1]
```diff
@@ -3,12 +3,32 @@ Command line interface for controller manager
...
+optional arguments:
+  -h, --help            show this help message and exit
+  --spin-time SPIN_TIME
+                        Spin time in seconds to wait for discovery (only applies when not using an already running daemon)
+  -c CONTROLLER_MANAGER, --controller-manager CONTROLLER_MANAGER
+                        Name of the controller manager ROS node
+  --include-hidden-nodes
+                        Consider hidden nodes as well
+```
+
```
Comment (destogl) [inline ros2controlcli/README.md:26]: Isn't this deprecated. If so please remove.

File: ros2controlcli/doc/index.rst [hunk 2]
```diff
@@ -22,3 +22,230 @@ Deprecated commands are
     - ros2 control configure_start_controller
     - ros2 control load_configure_controller
     - ros2 control load_start_controller
+
+configure_controller
+--------------------
```
Comment (destogl) [inline ros2controlcli/doc/index.rst:27]: Same comment/question as above.

**Review by destogl — APPROVED** [review]
Very good, just two minor comments.

Also, since GH can also display rst files, would it make sense to remove `README.md` and create a new link from `README.rst` to `ḋoc/index.rst` in this way we would have to manage only one file with description and not two.

**Comment by mahaarbo** [comment]
> Also, since GH can also display rst files, would it make sense to remove `README.md` and create a new link from `README.rst` to `ḋoc/index.rst` in this way we would have to manage only one file with description and not two.

Sounds like a good idea, and it would be much easier to maintain, should that be a symlink or just a link in the `README.MD`?

File: ros2controlcli/README.md (hunk 1 above)
Comment (v-lopez) [reply ros2controlcli/README.md:26]: +1 since we're breaking command API anyway, might as well remove the deprecated ones.

**Review by v-lopez — CHANGES_REQUESTED** [review]

**Comment by destogl** [comment]
> Sounds like a good idea, and it would be much easier to maintain, should that be a symlink or just a link in the `README.MD`?

I think you have to name it `README.rst` to render it properly by GitHub because there are subtle differences between RestructuredText and Markdown.


@bmagyar, @Karsten1987, @mahaarbo, @v-lopez : What do you also think about renaming the flags in `switch_controller` command:

Current:
``` 
switch_controller --start-controllers
```

New:
```
switch_controller --start
```

File: ros2controlcli/README.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Read the `Docs`_ for example usage and command line arguments.
```
Comment (destogl) [inline ros2controlcli/README.rst:1]: You can do either this or does it work with a symlink? So we have only one physical file? `ln -s` command in terminal.

Comment (mahaarbo) [reply ros2controlcli/README.rst:1]: I first thought about symlink as well, but isn't it a bit problematic when the codebase should support various filesystems/OSes? Or has it improved in recent years?

Also tried `.. include`, but that github markup does not support includes.

File: ros2controlcli/ros2controlcli/verb/load_controller.py [hunk 4]
```diff
@@ -21,15 +21,15 @@
...
+class LoadControllerVerb(VerbExtension):
     """Load a controller in a controller manager."""
 
     def add_arguments(self, parser, cli_name):
         add_arguments(parser)
         arg = parser.add_argument('controller_name', help='Name of the controller')
         arg.completer = ControllerNameCompleter()
         arg = parser.add_argument(
-            '--state',
+            '--set_state',
```
Comment (destogl) [inline ros2controlcli/ros2controlcli/verb/load_controller.py:32]: why to change this? and why not use the following?

```suggestion
            '--set-state',
```

**Review by bmagyar — APPROVED** [review]
After @destogl 's comments are resolved, I'm happy to go ahead with this.
Thanks for taking the time!

**Comment by destogl** [comment]
@mahaarbo  After this is merged, can you open a PR removing deprecated verbs:
- load_configure
- load_start
- configure_start

File: ros2controlcli/ros2controlcli/verb/load_controller.py (hunk 4 above)
Comment (mahaarbo) [reply ros2controlcli/ros2controlcli/verb/load_controller.py:32]: It was me misunderstanding https://github.com/ros-controls/ros2_control/issues/362 and then reverting wrongly.

**Comment by bmagyar** [comment]
Extending :cookie: to all involved, merging...

**Pull request merged**